_posts/en/newsletters/2021-01-13-newsletter.md

```diff
@@ -0,0 +1,213 @@
+    messages such as `addr` messages used for IP address announcements.
+    The `disabletx` negotiation persists for the lifetime of the
+    connection, allowing peers to use different limits for disabled
+    relay connections.
+
+- **Bech32m** Pieter Wuille [posted][wuille bech32m post] to the
+  Bitcoin-Dev mailing list a [draft BIP][bech32m bip] for a modified
+  version of the [bech32][topic bech32] algorithm that increases the
+  chance that the accidental addition or removal of characters in one of
+  these *bech32m* addresses will be detected.  If no problems are found
+  with the proposal, it is expected that bech32m addresses will be used
+  for [taproot][topic taproot] addresses and possibly future new script
+  upgrades.
+
+- **LN dual funding anti UTXO probing:** a long-term goal for LN has
+  been dual funding, the ability to open a new channel with funds from
+  both the node initiating the channel and the peer receiving their
+  request.  This will allow payments to travel in either direction in
+  the channel from the moment it's fully opened.  Before the initiator
+  can sign the dual funding transaction, they need the identities
+  (outpoints) of all of the UTXOs the other party wants to add to the
+  transaction.  This creates a risk that an abuser will attempt to
+  initiate dual-funded channels with many different users, learn their
+  UTXOs, and then refuse to sign the funding transaction---harming those
+  users' privacy at no cost to the abuser.
+
+    This week, Lloyd Fournier [posted][fournier podle] to the Lightning-Dev
+    mailing list an evaluation of two previous proposals to deal
+    with this problem, [one][zmn podle] using Proofs of Discrete Log
+    Equivalency (PoDLEs, see [Newsletter #83][news83 podle]) and the
+    [other][darosior sighash] using dual funding transactions
+    half-signed with `SIGHASH_SINGLE|SIGHASH_ANYONECANPAY`.  Fournier
+    extended the previous half-signed proposal and then provided his own
+    proposal that was equivalently effective but simpler.  The new
+    proposal has the initiator create and sign (but not broadcast) a
+    transaction that spends their UTXO back to themselves.  They give
+    this to the other party as a proof of good faith---if the initiator
+    later fails to sign the actual funding transaction, the respondent
+    can broadcast the good-faith transaction, forcing the initiator to
+    pay an onchain fee.  Fournier concludes his post with a summary of
+    the tradeoffs between the different approaches.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Add unit testing of node eviction logic][review club
+#20477] is a PR ([#20477][Bitcoin Core #20477]) by
+[practicalswift][practicalswift] to improve test coverage of Bitcoin Core's peer
```

Comment:
I don't think a link to a github profile makes much sense. I don't recall seeing much linking to individuals much in the past.

```suggestion
practicalswift to improve test coverage of Bitcoin Core's peer
```

Reply:
true